fe/fe-core/src/main/java/com/starrocks/system/ComputeNode.java

```diff
@@ -397,13 +397,20 @@ public boolean handleHbResponse(BackendHbResponse hbResponse) {
             this.lastUpdateMs = hbResponse.getHbTime();
             if (!isAlive.get()) {
                 isChanged = true;
-                this.lastStartTime = hbResponse.getHbTime();
+                // From version 2.5 we not use isAlive to determine whether to update the lastStartTime 
+                // This line to set 'lastStartTime' will be removed in due time
+                this.lastStartTime = hbResponse.getHbTime(); 
                 LOG.info("{} is alive, last start time: {}", this.toString(), hbResponse.getHbTime());
                 this.isAlive.set(true);
             } else if (this.lastStartTime <= 0) {
                 this.lastStartTime = hbResponse.getHbTime();
             }
 
+            if (hbResponse.isSetFirstHeartbeat() && hbResponse.isFirstHeartbeat()) {
+                this.lastStartTime = hbResponse.getHbTime();
+                isChanged = true;
+            }
```

Comment:
Could you add a more detailed description of the problem you are trying to resolve? It seems to me that if the leader of FE changes after a BE reboots, this `lastStartTime` will be inaccurate again.

Reply:
isFirstHeartbeat is a property in be, fe change leader has no effect on this pr’s logic

Reply:
Will the new leader go to line 410? If not, then the new leader won't get the accurate BE reboot time.

Reply:
done. PTAL

Reply:
use reboot time instead PTAL